Main.tex

```diff
@@ -82,48 +52,46 @@ \subsection{Ethik}\label{subsection:ethicsdefinition}
+Im Gegensatz zu bisherigen AIs können generative artifical intelligence (GAI) neue Daten generieren, anstatt wie bisher Daten zu analysieren oder basierend auf gegebenen Daten zu handeln. Außerdem werden ihnen zum Trainieren enorme Datenmengen, wie der komplette Inhalt von Wikipedia, zur Verfügung gestellt. Diese AIs sind daher erst mit steigender Rechenkapazität möglich geworden. GAIs modellieren Netze aus miteinander verbundenen Daten in verschiedenen multimedialen Formaten. Demnach bestünden beispielsweise Verbindungen zwischen dem Wort Panda, ein Bild von einem Panda und einem Video von einem Panda. Dadurch können GAIs ein beliebiges Eingabeformat in ein beliebiges Ausgabeformat übersetzen. Thema dieser Arbeit sind image generating 
 artifical intelligence (IGAI) oder auch text-to-image AIs, eine Unterkategorie von GAIs. Diese AIs können nach Eingabe von natürlichsprachlichem Text Bilder erzeugen.
 
 
 Um GAIs zu trainieren, werden drei Arten von Machine Learning verwendet: überwachtes Lernen, unüberwachtes Lernen und verstärktes Lernen. 
 Beim überwachten Lernen werden gekennzeichnete Datensätze verwendet, um Vorhersagen zu treffen. Die gekennzeichneten Daten werden als Trainingsdaten verwendet und nach dem Ausführen des Algorithmus wird überprüft, ob die getroffene Vorhersage der Kennzeichnung entspricht.
 Beim unüberwachten Lernen wird dem Algorithmus nicht gekennzeichnete Daten zugeführt. Hierbei wird nicht versucht eine Vorhersage zu treffen, sondern Schlussfolgerungen oder Zusammenhänge zwischen den Daten zu finden.
-Beim verstärkten Lernen wird ein System aus Belohnung und Strafen verwendet. Macht der Algorithmus wenige Fehler ist die Belohnung groß und die Strafe klein, macht der Algorithmus viele Fehler ist die Belohnung klein und die Strafe groß. Dadurch kann das Finden des 
+Beim verstärkten Lernen wird ein System aus Belohnung und Strafen verwendet. Macht der Algorithmus wenige Fehler ist die Belohnung groß und die Strafe klein, macht der Algorithmus viele Fehler ist die Belohnung klein und die Strafe groß. Dadurch kann die Identifizierung und Erlernung des 
 optimalen Verhaltens automatisiert werden.
 
 \chapter{Durchführung}
-Verschiedene Formen der Voreignenommenheit von AI lassen sich durch unterschiedliche Methoden verhindern. Diese lassen sich allgemein in 
-Technische und Nicht-Technische Methoden unterteilen. Nachfolgen werden verschiedene Methoden erläutert.
+Verschiedene Formen der Voreingenommenheit von AI lassen sich durch unterschiedliche Methoden verhindern. Nachfolgend werden die Methoden erläutert, wobei sie in technische und nicht-technische unterteilt werden.
+
 \section{Nicht-Technische Methoden}
-Nicht-Technische Methoden sind solche, die nicht primär auf Technische Lösungen setzen.
+nicht-technische Methoden sind solche, die nicht primär auf technische Lösungen setzen.
```

Comment:
was ein Humbug, das war mein Vorschlag dazu: "Nicht-technische Ansätze sind hierbei solche, die nicht direkt in die Funktionsweise der IGAI eingreifen. Diese Methoden liegen damit nicht in der Verantwortung des Entwicklers der IGAI, sondern in der Verantwortung des Anbieters oder des Anwenders der IGAI."

Reply:
wobei der zweite Satz da eigentlich auch so nicht mehr rein gehört

Reply:
den Vorschlag finde ich soweit gut, bis auf den zweiten Satz.

Reply:
Naja wenn mit dem Entwickler nur der Hans am Laptop gemeint ist, der den Code schreibt passt das...